Runtimes/Core/CMakeLists.txt

```diff
@@ -0,0 +1,83 @@
+# Notes:
+#
+# The Demangling library uses `#if SWIFT_HAVE_CRASHREPORTERCLIENT` while the
+# runtime library uses `#ifdef SWIFT_HAVE_CRASHREPORTERCLIENT` to toggle that
+# functionality. When building the demangling library, the macro should be set
+# to 0 or 1 to indicate the presence of the crashreporter.
+# When building the runtime library, the existence of the macro indicates the
+# presence of the crashreporter.
```

Comment:
Can we set it to 1 when enabled and un-set it when disabled? That should work for both.

Reply:
As far as I can tell, C/C++ defines it so that if a `-DFOO` is passed without a value, it's given the value of `1`, and `#if FOO` will see undefined an `0` the same way. We might be able to get away with just setting it. I'll have to compare the symbol lists though.